src/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -64,41 +66,202 @@ public static void CreateInstance_ConstructorWithParamsParameter()
         }
 
         [Fact]
-        public static void CreateInstance_Invalid()
+        public void CreateInstance_ValueTypeWithPublicDefaultConstructor_Success()
         {
-            Type nullType = null;
-            AssertExtensions.Throws<ArgumentNullException>("type", () => Activator.CreateInstance(nullType)); // Type is null
-            AssertExtensions.Throws<ArgumentNullException>("type", () => Activator.CreateInstance(null, new object[0])); // Type is null
+            // Test caching behaviour by activating multiple times.
```

Comment:
What caching behavior is this validating / testing?

Reply:
https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/RtType.cs#L4768-L4781 for example, but I clarified this comment